tests/test_lldpSyncDaemon.py

```diff
@@ -87,3 +87,17 @@ def test_sync_roundtrip(self):
     def test_timeparse(self):
         self.assertEquals(lldp_syncd.daemon.parse_time("0 day, 05:09:02"), make_seconds(0, 5, 9, 2))
         self.assertEquals(lldp_syncd.daemon.parse_time("2 days, 05:59:02"), make_seconds(2, 5, 59, 2))
+
+    def test_parse_mgmt_ip(self):
+        parsed_update = self.daemon.parse_update(self._json)
+        mgmt_ip_str = parsed_update['local-chassis'].get('lldp_loc_man_addr')
+        json_chassis = json.dumps(self._json['lldp_loc_chassis']['local-chassis']['chassis'])
+        chassis_dict = json.loads(json_chassis)
+        json_mgmt_ip = chassis_dict.values()[0]['mgmt-ip']
+        if isinstance(json_mgmt_ip, list):
```

Comment:
>isinstance [](start = 11, length = 10)

Thanks for add this test case!
I notice there is no mock data for multiple-ip. Could you add one or modify one in /tests/mock_tables/LLDP_ENTRY_TABLE.json?

Reply:
Do you hope that mock data about LLDP_LOC_CHASSIS? If yes, I will add another file. Thanks.

Reply:
Yes. Either modify existing file or add another one.

Reply:
Hi @qiluo-msft , I modified /tests/mock_tables/LLDP_ENTRY_TABLE.json and added 'test_loc_chassis' test case. Please help to review code again. Thanks very much.
> 
    sonic-dbsyncd$ pytest -v
    ================================================= test session starts ==================================================
    platform linux2 -- Python 2.7.15rc1, pytest-3.3.2, py-1.5.2, pluggy-0.6.0 -- /usr/bin/python2
    cachedir: .cache
    rootdir: /mnt/d/lldp_syncd/sonic-dbsyncd, inifile:
    collected 7 items   

    tests/test_lldpSyncDaemon.py::TestLldpSyncDaemon::test_loc_chassis PASSED                                        [ 14%]
    tests/test_lldpSyncDaemon.py::TestLldpSyncDaemon::test_parse_json PASSED                                         [ 28%]
    tests/test_lldpSyncDaemon.py::TestLldpSyncDaemon::test_parse_mgmt_ip PASSED                                      [ 42%]
    tests/test_lldpSyncDaemon.py::TestLldpSyncDaemon::test_parse_short PASSED                                        [ 57%]
    tests/test_lldpSyncDaemon.py::TestLldpSyncDaemon::test_parse_short_short PASSED                                  [ 71%]
    tests/test_lldpSyncDaemon.py::TestLldpSyncDaemon::test_sync_roundtrip PASSED                                     [ 85%]
    tests/test_lldpSyncDaemon.py::TestLldpSyncDaemon::test_timeparse PASSED                                          [100%] 

    =============================================== 7 passed in 0.23 seconds ===============================================